docs/user/FAQ.rst

```diff
@@ -413,6 +413,51 @@ the **greatest** score.
     best_net = grid_searcher.best_estimator_
     print(best_net.score(X, y))
 
+
+How can I set the random seed of my model?
+------------------------------------------
+
+skorch does not provide an unified way for setting the seed of your model.
+You can utilize the `numpy <https://pytorch.org/docs/stable/generated/torch.manual_seed.html>`__
+and `torch <https://pytorch.org/docs/stable/generated/torch.manual_seed.html>`__
+seeding interfaces to set random seeds before model initialization and will
+get consistent results if you do not employ other libraries that introduce randomness.
```

Comment:
Maybe add a word about `torch.cuda.manual_seed`?

Reply:
Yep, since it is called if needed I explained this behavior.